sntpc/Cargo.toml

```diff
@@ -26,15 +26,16 @@ sync = ["dep:miniloop"]
 utils = ["std", "dep:chrono", "chrono/clock"]
 log = ["dep:log"]
 std-socket = []
-embassy-socket = ["dep:embassy-net"]
 tokio-socket = ["dep:tokio"]
 defmt = ["dep:defmt", "embassy-net/defmt"]
+embassy-socket = ["dep:embassy-net"]
+embassy-socket-ipv6 = ["embassy-socket", "embassy-net?/proto-ipv6"]
 
 [dependencies]
 log = { version = "~0.4", optional = true }
 chrono = { version = "~0.4", default-features = false, optional = true }
 miniloop = { version = "~0.3", optional = true }
-embassy-net = { version = ">=0.5", features = ["udp", "proto-ipv4", "proto-ipv6", "medium-ip"], optional = true }
+embassy-net = { version = "0.6", features = ["udp", "proto-ipv4", "medium-ip"], optional = true }
```

Comment:
Would you please tell, why we need to bump `embassy-net` version to `0.6`? I checked the embassy's `UdpSocket` implementation and there seems to be no breaking changes since `0.5+`. So, I would prefer to leave more permissive requirements.

Reply:
I'm not that familiar with what `>=` actually does, but I'm guessing it will allow cargo to update to 0.6 or 0.7 etc in the future? If so, I don't think this is a good idea, just because the socket didn't change in 0.6 doesn't mean it won't later, and if it does this crate will break the second the end user runs `cargo update`, even if the end user hasn't updated to the newer embassy-net version.

Reply:
In this case it would be up to the user to specify a specific version in his app/library `Cargo.toml`. Currently, the `sntpc` should work properly with `0.5` and `0.6` versions of the `embassy-net`. Restricting it to only `0.6` breaks examples in this repository which cannot resolve some of their example specific dependencies with the updated `embassy-net == 0.6` rule.